build/azure-pipelines/win32/sql-product-build-win32.yml

```diff
@@ -37,7 +37,7 @@ steps:
     inputs:
       azureSubscription: 'ClientToolsInfra_670062 (88d5392f-a34f-4769-b405-f597fc533613)'
       KeyVaultName: ado-secrets
-      SecretsFilter: 'github-distro-mixin-password'
+      SecretsFilter: 'github-distro-mixin-password,standalone-extensions-uri'
```

Comment:
What is this URI?

In general extension packaging stuff like this should probably be done in a gulp task - that's a lot easier to work with and keeps these scripts from getting cluttered with too much extra stuff.

Reply:
We already have a task to package in external extensions so it seems like extending that as needed would help keep the logic all together : https://github.com/microsoft/azuredatastudio/blob/main/build/azure-pipelines/mixin.js#L36

Reply:
(that pulls from https://github.com/microsoft/azuredatastudio/blob/main/product.json#L76 if it isn't clear)

Reply:
Thanks @Charles-Gagnon for the pointer on the gulp task! Will move things over there. As far as the market place, we probably won't use the marketplace gallery as what we have is internal only, so we don't want that exposed.

Reply:
If you're doing that please use a clearer name - maybe something like `external-extensions-uri`

What extensions are these anyways?

Reply:
Livesite extension, Simple PR